opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -694,8 +812,15 @@ def __init__(
         sampler: sampling.Sampler = trace_api.sampling.ALWAYS_ON,
         resource: Resource = Resource.create_empty(),
         shutdown_on_exit: bool = True,
+        active_span_processor: Union[
+            SynchronousMultiSpanProcessor, ConcurrentMultiSpanProcessor
+        ] = None,
     ):
-        self._active_span_processor = MultiSpanProcessor()
+        self._active_span_processor = (
+            SynchronousMultiSpanProcessor()
```

Comment:
this could be `self_active_span_processor = active_span_processor or SynchronousMultiSpanProcessor()`, taking advantage of Python's falsy values.